docs/modules/ROOT/pages/_partials/creating-fuse-integration.adoc

```diff
@@ -16,15 +16,61 @@ endif::location[]
 
 . Log in to the link:{fuse-url}[Red Hat Fuse Online, window="_blank"] console.
 
+. Select *Customizations* from the left hand menu.
+
+. Select the *Create API Connector* button to start the *API Client Connector* wizard.
+
+. When prompted to *Upload OpenAPI Specification*, select *Use a URL*.
+.. Enter the following in the URL field. `http://{spring-boot-url}/v2/api-docs`
+.. Click *Next*
+
+. When prompted with *Review Actions*, select *Next*.
+
+. When prompted with *Specify Security*, select *Next*.
+
+. When prompted with *Review/Edit Connector Details*
+.. Enter the following in the *Connector Name* field, `Walkthrough One CRUD Connector`.
+.. Click *Create API Connector*
+
+. Select *Connections* from the left hand menu.
+
+. Select the *Create Connection* button to start the *Create Connection* wizard.
+
+. When prompted with *Select Connector*, select *Walkthrough One CRUD Connector*.
+
+. When prompted with *Walkthrough One CRUD Configuration*, click *Next*.
+
+. When prompted with *Add Connection Details*
+.. Enter the following in the *Connection Name* field, `Walkthrough One CRUD App`
+.. Click *Create*
+
+. Select *Connections* from the left hand menu.
+
+. Select the *Create Connection* button to start the *Create Connection* wizard.
+
+. When prompted with *Select Connector*, select *AMQ Message Broker*
+
+. When prompted with *AMQ Message Broker Configuration*
+.. Enter the following into the *Broker URL* field, `tcp://{messaging-url}:61616`
```

Comment:
Could we change `messaging-url` to `messaging-broker-url`?

Is there being no defaults available ok for now?